install/kots/manifests/kots-preflight.yaml

```diff
@@ -106,21 +106,21 @@ spec:
         checkName: Total CPU Cores in the cluster
         outcomes:
           - fail:
-              when: "sum(cpuCapacity) < 2"
+              when: "min(cpuCapacity) < 2"
               message: The cluster must contain at least 2 cores
           - warn:
-              when: "sum(cpuCapacity) < 4"
+              when: "min(cpuCapacity) < 4"
               message: The cluster must contain at least 4 cores
           - pass:
               message: There are at least 4 cores in the cluster
     - nodeResources:
         checkName: Total memory in the cluster
```

Comment:
These aren't checking for the total's of the cluster anymore, right? Should be named around the nodes?

Reply:
@MrSimonEmms Any thoughts on this?

Reply:
@corneliusludmann sorry, I missed this.

As I see it, we can do one of two things:
1. put a filter in so that the workspace nodes act as above and the "meta" nodes have no `warn` section
2. leave as-is

Reply:
@MrSimonEmms I don't think my question was around `meta` vs `workspace` nodes. It was just to update the `checkName` to reflect it to be more about induvidual node's min requirement than the total. 🤔

Reply:
Oh right, yes I see